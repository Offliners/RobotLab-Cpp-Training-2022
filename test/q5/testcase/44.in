14*20/6=*(19-4*7+18*(16*16-8))/17+6*15/(14-8/14/16-11)/6+17-14+9*19-3+2-19=+1-9+2*5/7%8=-16*(20-20)+(13+15)+14*3%4=+(19+11)*4=/(5-18/14+15+12-15+8+(1/1/2))/7+5/15/1/(16+3)/13/14/14+12=*(14/(9-10))/18-(20*1-8)=/9-18-17-9=
