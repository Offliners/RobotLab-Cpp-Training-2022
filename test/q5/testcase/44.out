38
76
-200
-841
